dependency-setup.sh

```diff
@@ -21,7 +21,7 @@ else
     (cd $JSR308 && git clone -b $BRANCH --depth 1 https://github.com/"$REPO_SITE"/checker-framework-inference.git)
 fi
 
-(cd $JSR308/checker-framework-inference && ./.travis-build-without-test.sh)
+(cd $JSR308/checker-framework-inference && ./.travis-build-without-test.sh && ./gradlew testLibJar)
```

Comment:
Should the `./gradlew testLibJar` part go into `.travis-build-without-test.sh`?

Reply:
Thanks for pointing out! I will do it.